tools/codeformat.py

```diff
@@ -128,23 +135,19 @@ def fixup_c(filename):
 
 
 def main():
-    cmd_parser = argparse.ArgumentParser(description="Auto-format C and Python files.")
+    cmd_parser = argparse.ArgumentParser(description="Auto-format C and Python files -- to be used via pre-commit only.")
```

Comment:
why change this?

Reply:
During my work to fix the pre-commit process, I repeatedly had trouble because invoking codeformat.py directly did things that were undesirable.  Do you see it as important to your workflow to invoke codeformat.py directly? (anyway, it's just advice, we don't stop you)

Reply:
Not important to my workflow but I didn't encounter any issues running `codeformat.py` directly.